app/assets/javascripts/govuk_publishing_components/components/step-by-step-nav.js

```diff
@@ -29,6 +30,7 @@
       var $showOrHideAllButton;
 
       uniqueId = $element.data('id') || false;
+      sessionStoreLink = sessionStoreLink + "_" + uniqueId;
```

Comment:
What happens here if `uniqueId` is `false`?

Reply:
@leenagupte that's an excellent question - I hadn't considered that possibility. The functionality depends upon that, so I'll have a think about how this should be handled.

Reply:
I've put a check in so if `uniqueId` is false `sessionStoreLink` is just 'govuk-step-nav-active-link', and added a note to the documentation.